backend-api/routes/api.php

```diff
@@ -12,7 +11,29 @@
 | is assigned the "api" middleware group. Enjoy building your API!
 |
 */
-
 Route::middleware('auth:api')->get('/user', function (Request $request) {
     return $request->user();
 });
+
+Route::get('users/', 'AuthController@me');
+Route::get('users/', 'AuthController@me');
+Route::post('/user', 'AuthController@createUser');
+Route::put('users/{id}', 'AuthController@me');
```

Comment:
You are using a `PUT` and `me` should be a get. A `PUT` of /me should edit the user (`PUT` is always an edition).

Reply:
It doesn't make sense indeed. We deleted that route.